examples/bpmn-utlis.js

```diff
@@ -0,0 +1,577 @@
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="532" y="343" width="68" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__0_intermediate_catch_event_2_1" bpmnElement="intermediate_catch_event_2_1">
+        <dc:Bounds x="740" y="97" width="32" height="32" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="725" y="132" width="60" height="30" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Trisotech.Visio__0_task_2_2" bpmnElement="task_2_2">
+        <dc:Bounds x="601" y="79" width="83" height="68" />
+        <bpmndi:BPMNLabel />
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNEdge id="Trisotech.Visio__0_flow_2_1" bpmnElement="flow_2_1">
+        <di:waypoint x="185" y="219" />
+        <di:waypoint x="228" y="219" />
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="Trisotech.Visio__0_flow_2_2" bpmnElement="flow_2_2">
+        <di:waypoint x="311" y="219" />
+        <di:waypoint x="400" y="219" />
+        <bpmndi:BPMNLabel />
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="Trisotech.Visio__0_flow_2_3" bpmnElement="flow_2_3">
+        <di:waypoint x="442" y="219" />
+        <di:waypoint x="550" y="219" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="465" y="194" width="42" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="Trisotech.Visio__0_flow_2_4" bpmnElement="flow_2_4">
+        <di:waypoint x="421" y="198" />
+        <di:waypoint x="421" y="113" />
+        <di:waypoint x="601" y="113" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="434" y="123" width="52" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="Trisotech.Visio__0_flow_2_5" bpmnElement="flow_2_5">
+        <di:waypoint x="582" y="219" />
+        <di:waypoint x="642" y="219" />
+        <di:waypoint x="642" y="147" />
+        <bpmndi:BPMNLabel />
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="Trisotech.Visio__0_flow_2_6" bpmnElement="flow_2_6">
+        <di:waypoint x="421" y="240" />
+        <di:waypoint x="421" y="317" />
+        <di:waypoint x="550" y="317" />
+        <bpmndi:BPMNLabel>
+          <dc:Bounds x="429" y="293" width="15" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="Trisotech.Visio__0_flow_2_7" bpmnElement="flow_2_7">
+        <di:waypoint x="772" y="113" />
+        <di:waypoint x="834" y="113" />
+        <bpmndi:BPMNLabel />
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="Trisotech.Visio__0_flow_2_8" bpmnElement="flow_2_8">
+        <di:waypoint x="684" y="113" />
+        <di:waypoint x="740" y="113" />
+        <bpmndi:BPMNLabel />
+      </bpmndi:BPMNEdge>
+    </bpmndi:BPMNPlane>
+  </bpmndi:BPMNDiagram>
+</semantic:definitions>
+        `;
+}
+
+
+// Other
+function getHorizontalBpmnDiagram() {
```

Comment:
:question: I suggest we state that it is a custom diagram to show fit capabilities

Reply:
Done